civihr_employee_portal/views/views_export/views_mydetails_contactdetails.inc

```diff
@@ -0,0 +1,89 @@
+<?php
```

Comment:
@powdevel why we are not using feature for this ?

Reply:
@hitesh-jain I wouldn't know for sure. I'm following current patterns in `civihr_employee_portal` as this module is not a feature (at least it is not listed as a feature), however it has exported views and nodes with custom methods to revert things like nodes, see `drush_civihr_employee_portal_refresh_node_export_files` which is used many times on `civihr_employee_portal.install`. Please correct me if I'm wrong.